### PR TITLE
Doc: Add missing configuration file directives

### PR DESCRIPTION
**Pull request by Berbe**

### CODE REVIEW

File: docs/Editing-Configuration-Files.md [hunk 1]
```diff
@@ -119,6 +120,7 @@ Here is a sample of the three basic types: respectively Boolean, Number and Stri
  * **rpc-host-whitelist-enabled:** Boolean (default = true. Added in v2.93)
  * **rpc-password:** String. You can enter this in as plaintext when Transmission is not running, and then Transmission will salt the value on startup and re-save the salted version as a security measure. **Note:** Transmission treats passwords starting with the character `{` as salted, so when you first create your password, the plaintext password you enter must not begin with `{`.
  * **rpc-port:** Number (default = 9091)
+ * **rpc-socket-mode:** String UNIX filesystem mode for the RPC UNIX socket
```
Comment (ckerr) [inline docs/Editing-Configuration-Files.md:123]:
```suggestion
 * **rpc-socket-mode:** String UNIX filesystem mode for the RPC UNIX socket (default: 0750; used when `rpc-bind-address` is a UNIX socket)
```

**Review by ckerr — COMMENTED** [review]
Nice catches! Thanks for documenting these!

**Comment by Berbe** [comment]
> Nice catches! Thanks for documenting these!

Glad to be of service!

Those catches actually started from another endavour: I noticed the directives used in a previous instance and the defaults generated from `4.0.0` were different, and I started spotting the differences.

My initial issue remains, which I'd like to fix but do not know exactly how, is that some (documented) directives are not part of the default configuration file generated on the `transmission-daemon` startup. Cf. #4822

**Review by ckerr — APPROVED** [review]

**Comment by ckerr** [comment]
Notes: Improved documentation on settings.json options.

**Pull request merged**